src/reporter/delete.py

```diff
@@ -34,11 +34,19 @@ def delete_entity(entity_id, type_=None, from_date=None, to_date=None):
 def delete_entities(entity_type, from_date=None, to_date=None,
                     drop_table=False):
     with translator_for(fiware_s()) as trans:
-        if drop_table:
```

Comment:
it looks like this file doesn't belong in this PR? is it a relic of the other pr you worked on to drop tables? is git playing tricks on you? :-)

Reply:
Hi @c0c0n3 , As I have same repository and different branches for all previously raised PR that is why I guess its showing this [PR ](https://github.com/orchestracities/ngsi-timeseries-api/pull/455/files) changes too.